### PR TITLE
feat: Added Search Feature and Updated Theme

### LINKED ISSUE #710: [BUG] - Table and Links Messed Up
**Describe the bug**
The table and links at https://3dmol.org/doc/tutorial-url.html#gsc.tab=0 are completely messed up

**To Reproduce**
- Go to https://3dmol.org/doc/tutorial-url.html
- Scroll down a bit and then you will see the error

**Expected behavior**
- Tables should be rendered properly

**Screenshots**
Screenshot 2023-08-16 at 1 14 31 AM

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- This PR adds a search feature and also updates to a new theme for the docs based upon js-docs
- Also, Fixes #710 

Note: Will demo the new things in our call tomorrow :)

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
@dkoes could you please review this MR? I've added a new tutorials page as well. Please do lmk if there are any changes to be done to it :)

File: doc.md [hunk 1]
```diff
@@ -15,20 +15,21 @@
...
-* support for pdb, sdf, mol2, xyz, and cube formats
-* parallelized molecular surface computation
-* sphere, stick, line, cross, cartoon, and surface styles
-* atom property based selection and styling
-* labels
-* clickable interactivity with molecular data
-* geometric shapes including spheres and arrows
+Features include:
+
+1. Support for pdb, sdf, mol2, xyz, and cube formats
```
Comment (dkoes) [inline doc.md:22]: This should not be an ordered list, it is unordered.

File: jsdoc.conf.json [hunk 2]
```diff
@@ -2,44 +2,58 @@
...
+            "homepageTitle": "3Dmol.js",
+            "title": "3Dmol.js",
+            "menu": [
+                {
+                    "title": " Documentation",
+                    "link": "https://3dmol.org/doc/index.html"
+                },
+                {
+                    "title": "Tutorials",
+                    "link": "/doc/tutorial-home.html"
```
Comment (dkoes) [inline jsdoc.conf.json:49]: this link does not work - you need relative links

File: tutorials/embeddable.md [hunk 3]
```diff
@@ -14,23 +14,25 @@ The code below is all that is needed to create the displayed viewer.
...
- - **data-backgroundcolor** The background color (default black).
- - **data-backgroundalpha** The background alpha (default opaque: 1.0).
- - **data-select** The value is an {@link AtomSpec} selection specification.
- - **data-style** The value is a style specification.
- - **data-surface** A surface style specification.
- - **data-labelres** A residue label style specification.
- - **data-zoomto** An {@link AtomSpec} selection specification to zoom to.
- - **data-spin** If set will spin the model using {@link $3Dmol.GLViewer#spin}. Can specify axis and speed (e.g. data-spin='axis:z;speed:10')
- - **data-ui** If set will show the UI for the viewer.
+ 1. **data-pdb** The value describes a PDB ID to be loaded into the viewer.
```
Comment (dkoes) [inline tutorials/embeddable.md:17]: This should not be an ordered list, it is unordered.

File: tutorials/home.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+This guide helps you with learning how to,
+
+1. [Use 3Dmol within your code](tutorial-code.html)
```
Comment (dkoes) [inline tutorials/home.md:4]: This should not be an ordered list, it is unordered.

**Review by dkoes — COMMENTED** [review]
Please make sure there are no broken links.

File: doc.md (hunk 1 above)
Comment (adithyaakrishna) [reply doc.md:22]: The theme is not supporting unordered lists as of now, I have asked the maintainer of the same for more info

Comment (adithyaakrishna) [reply doc.md]: This is fixed, hence resolving this :)

File: jsdoc.conf.json [hunk 5]
```diff
@@ -2,44 +2,71 @@
...
+        "recurse": true,
+        "verbose": true,
+        "template": "./node_modules/clean-jsdoc-theme",
+        "copyright": "3Dmol.js © 2019-2023",
+        "theme_opts": {
+            "default_theme": "light",
+            "includeFilesListInHomepage": true,
+            "homepageTitle": "3Dmol.js",
+            "displayModuleHeader": true,
+            "footer":"3Dmol.js © 2019-2023 | <a href='https://pitt.edu/' target='_blank'>University of Pittsburg</a> | <a href='https://github.com/3dmol/3Dmol.js/blob/master/LICENSE' target='_blank'>License</a>",
```
Comment (dkoes) [inline jsdoc.conf.json:41]: Pittsburgh has an h at the end.

**Review by dkoes — CHANGES_REQUESTED** [review]

File: jsdoc.conf.json (hunk 5 above)
Comment (adithyaakrishna) [reply jsdoc.conf.json]: @dkoes Oops! My bad, thanks for pointing it out :)

**Comment by adithyaakrishna** [comment]
@dkoes Have fixed the typo with the above commit :)

**Review by dkoes — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
Lessgoooo 🔥

**Pull request merged**